crates/ruma-serde/src/base64.rs

```diff
@@ -9,6 +9,7 @@ pub struct Base64<B = Vec<u8>> {
     bytes: B,
 }
 
+// See https://github.com/matrix-org/matrix-doc/issues/3211.
```

Comment:
Can you remove the trailing `.`? It will often end up being interpreted as part of the URL by editors and stuff. (just happened to me here on GitHub w/ Refined GitHub extension)